org.metaborg.spoofax.shell.console/src/main/java/org/metaborg/spoofax/shell/client/console/ConsoleReplModule.java

```diff
@@ -48,15 +51,15 @@ protected void configure() {
      *            The {@link InputStream}.
      * @param out
      *            The {@link OutputStream}.
-     * @return a {@link ConsoleReader} with the given streams.
+     * @return a {@link jline.console.ConsoleReader} with the given streams.
      * @throws IOException
      *             When an IO error occurs upon construction.
      */
     @Provides
     @Singleton
-    protected ConsoleReader provideConsoleReader(@Named("in") InputStream in,
-                                                 @Named("out") OutputStream out)
-                                                     throws IOException {
-        return new ConsoleReader(in, out);
+    protected jline.console.ConsoleReader provideConsoleReader(@Named("in") InputStream in,
```

Comment:
Then it's clear that it's a library, not from us. But I can turn it back if you want.